bundles/org.openhab.binding.nuvo/README.md

```diff
@@ -473,12 +475,57 @@ A complete XML string for the desired menu is then stored in the `menuXmlSrcN` c
     <item>menu3 x</item>
     <item>menu3 y</item>
 </topmenu>
+<topmenu text="Turn off other zones"/>
 ```
 
 When a menu item is selected, the text of the topmenu item and sub menu item (if applicable) will be sent to the button channel in a pipe delimited format.
 For example, when item `menu1 b` is selected, the text `Top menu 1|menu1 b` will be sent to the button channel.
 When the item `Top menu 2` is selected the text sent to the button channel will simply be `Top menu 2` since this menu item does not have any sub menu items.
 
+### Rule to trigger an action based which keypad zone where a button was pressed or menu item selected
+
+By using the `system#button_press` channel it is possible to trigger an action based on which keypad zone was used to send the action.
+This channel appends the zone number and a comma before the button action or menu item selection.  
+
+For example if the Play/Pause button is pressed on Zone 7, the channel will display: `7,PLAYPAUSE`  
+Also if a menu item from a custom menu was selected, ie: `Top menu 1` on Zone 5, the channel will display: `5,Top menu 1`  
+
+The functionality can be utilized to create very powerful rules as demontrated below. The following rule triggered from a menu item turns off all zones except for the zone that triggered the rule.
+
+nuvo-turn-off-all-but-caller.rules:
+
+```
+rule "Turn off all zones except caller zone"
+when
+    Item nuvo_system_button_press received update
+then
+    var callerZone = nuvo_system_button_press.state.toString().split(",").get(0)
+    var button = nuvo_system_button_press.state.toString().split(",").get(1)
```

Comment:
It is always more reliable to use `newState` rather than `nuvo_system_button_press.state` in that context.

Reply:
updated and tested as working.